corehq/apps/groups/templates/groups/all_groups.html

```diff
@@ -35,6 +35,16 @@
 {% endblock %}
 
 {% block main_column %}
+{% if needs_to_downgrade_locations %}
+    <div>
+        <p>
+            {% trans "This project no longer has access to the Organization feature." %}
+            <a href="{% url "downgrade_locations" domain %}">
+                {% trans "Click here to fix this." %}
```

Comment:
Got it. I really don't like the empty page + extra click, but see the point about doing the location action in the context of the locations page.